src/inc/msquic.h

```diff
@@ -824,10 +824,9 @@ void
 #define QUIC_PARAM_GLOBAL_VERSION_SETTINGS              0x01000007  // QUIC_VERSION_SETTINGS
 #endif
 #define QUIC_PARAM_GLOBAL_LIBRARY_GIT_HASH              0x01000008  // char[64]
-#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
 #define QUIC_PARAM_GLOBAL_EXECUTION_CONFIG              0x01000009  // QUIC_EXECUTION_CONFIG
-#endif
 #define QUIC_PARAM_GLOBAL_TLS_PROVIDER                  0x0100000A  // QUIC_TLS_PROVIDER
+#define QUIC_PARAM_GLOBAL_DATAPATH_FEATURES             0x0100000B  // uint32_t
```

Comment:
Let's put this is msquicp.h and have this private for now. I don't want a public API that depends on a type only in quic_datapath.h, which isn't public.